rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/ChargingRSpace.scala

```diff
@@ -58,7 +58,9 @@ object ChargingRSpace {
           patterns: Seq[BindPattern],
           continuation: TaggedContinuation
       ): F[Option[(TaggedContinuation, Seq[ListParWithRandom])]] =
-        space.install(channels, patterns, continuation)
+        noCharge {
+          space.install(channels, patterns, continuation)
+        }
```

Comment:
this looks a bit redundant, but it doesn't matter too much. On one hand I understand the intent to show it's not doing any charging, OTOH the compiler doesn't care... Let's leave it though :)